pkg/skaffold/deploy/label/labeller.go

```diff
@@ -68,3 +68,7 @@ func (d *DefaultLabeller) Labels() map[string]string {
 func (d *DefaultLabeller) RunIDSelector() string {
 	return fmt.Sprintf("%s=%s", RunIDLabel, d.Labels()[RunIDLabel])
 }
+
+func (d *DefaultLabeller) GetRunID() string {
```

Comment:
How about d.RunID? So as this func is no longer needed.

Reply:
i don't want `runID` itself to be accessible because anyone with a `DefaultLabeller` could modify it. it's safer this way